src/licenses/build.gradle.kts

```diff
@@ -73,6 +74,7 @@ val gatherBinaryLicenses by tasks.registering(GatherLicenseTask::class) {
     configuration(binaryDependencies)
     ignoreMissingLicenseFor.add(SpdxLicense.Apache_2_0.asExpression())
     defaultTextFor.add(SpdxLicense.MPL_2_0.asExpression())
+    outputs.cacheIf { true }
```

Comment:
Same comment as above. Do you know more about this task? is it a disk bound task where it just copies a bunch of license files or is there some more processing involved here to merge them or something like that?